Frontend/CO.CDP.OrganisationApp/Pages/Registration/OneLoginCallback.cshtml.cs

```diff
@@ -0,0 +1,43 @@
+using CO.CDP.OrganisationApp.Models;
+using CO.CDP.OrganisationApp.ServiceClient;
+using CO.CDP.Tenant.WebApiClient;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace CO.CDP.OrganisationApp.Pages.Registration;
+
+public class OneLoginCallback(
+    IOneLoginClient oneLoginClient,
+    ITenantClient tenantClient,
+    ISession session) : PageModel
+{
+    public async Task<IActionResult> OnGet()
+    {
+        var userInfo = await oneLoginClient.GetUserInfo()
+                    ?? throw new Exception("Unable to retrive user info"); // show error page?
```

Comment:
We should probably show an error and offer to retry. Can you add it to the backlog please?

Reply:
Created the the backlog task  - 
https://noticingsystem.atlassian.net/browse/DP-105